step-8/README.adoc

```diff
@@ -106,32 +106,18 @@ include::src/main/java/io/vertx/guides/wiki/database/WikiDatabaseServiceImpl.jav
 
 === Working with a database connection
 
-When direct queries do not fit (e.g. when multiple queries must participate in the same transaction), you can acquire a database connection from the pool.
-All you have to do is calling `rxGetConnection` on the `JDBCClient`:
+When direct queries do not fit (e.g. when using session variables/temporary tables), you can generate a reactive flow from a single `SQLConnection`:
```

Comment:
The previous wording of multiple queries in a transaction was very clear, while the wording of _session variables/temporary tables_) is not clear to me. Can you reformulate this?

Reply:
I hesitated on this one :stuck_out_tongue: 

The things is I would prefer not to encourage developers to manage transactions themselves with the Rxified API (instead of using the new transactions transformers and helpers). That's why I changed the example.

Maybe I'm too cautious...

Reply:
You are absolutely right, it's just that the sentence here needs clarification 😉 

> When direct queries do not fit (e.g. when using session variables/temporary tables)

↕️ This is where I'm not sure what you mean.

Reply:
@jponge as discussed off-list, I reworded the example to better explain why you need to work with the same connection